README.md

```diff
@@ -1,6 +1,172 @@
-The purpose of the [OpenMRS Cohort Module](https://wiki.openmrs.org/display/projects/Cohort+Module) is to manage Cohort related Data, Types, Attributes, Attribute Types and Members while using REST to allow external system integration. Cohort management can be performed one of two ways:
```

Comment:
Add an introductory paragraph about the cohort module, like it was before you modified the REAMDE